Minesweeper.py

```diff
@@ -43,68 +28,86 @@
                 print(board[y][x], end=" ")
         print()
 
-    # Print the game stats
+def print_stats():
     print("Moves:", moves)
     print("Flags:", flags)
     print("Mines found:", mines_found)
 
-    # Get the player's input
-    action = input("Enter your action (move, flag, or unflag): ")
-    direction = input("Enter the direction (up, down, left, or right): ")
-
-    # Update the player's position
+def update_player_pos(direction):
```

Comment:
There's an issue: an indentation error in the conditional statements could affect the flow of the `update_player_pos` function. ```diff -      if direction == "up": +    if direction == "up": ``` ---